db/migrate/20210426201823_change_product_types.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeProductTypes < ActiveRecord::Migration[6.1]
+  def up
+    execute <<-SQL
+      INSERT INTO product_types (title, created_at, updated_at)
+      VALUES ('Diapers', now(), now()), ('Menstrual hygiene', now(), now())
+    SQL
+  end
+
+  def down
+    execute <<-SQL
+      DELETE FROM product_types WHERE title = 'Diapers';
+      DELETE FROM product_types WHERE title = 'Menstrual hygiene'
+    SQL
```

Comment:
Use Rails model to create records

Reply:
Thank you for response. I made recommended changes.